### ISSUE TITLE
Change identifier variable used in LocationHierarchy and PractitionerDetails endpoints

### ISSUE THREAD

**Issue by lincmba** [onaio/fhir-gateway-extension#61]
In other fhir endpoints, the identifier variable used is `_id` however in LocationHierarchy and PractitionerDetails endpoints, the identifier variable used is `identifier`.
This should be changed to `_id` to be consistent with other fhir endpoints

**Comment by dubdabasoduba** [comment]
@lincmba I think the PractitionerDetails endpoint will still use the identifier. 

The endpoint is mostly called after logging in and at this time we only have the Practitioners Keycloak UUID. This UUID is stored on the `Practitioner.Identifier` in the FHIR resources. 

At the point of login, we do not have a way to know the `Practitioner.id` hence the use of the `identifier`

**Comment by dubdabasoduba** [comment]
I agree with changing the LocationHierarchy endpoint to use the `id` but the PractitionerDetails might have to remain the same unless we refactor the whole implementation

**Comment by ndegwamartin** [comment]
Also to be specific the **PractitionerDetail** endpoint uses `keycloak-uuid` as the GET request parameter i.e.

```
/PractitionerDetail?keycloak-uuid=<some uuid>
```
